test-data/unit/check-flags.test

```diff
@@ -1053,25 +1053,23 @@ always_false = BLAH, BLAH1
 [builtins fixtures/bool.pyi]
 
 [case testCheckDisallowAnyGenericsNamedTuple]
-# Crashes on new semantic analyzer: https://github.com/python/mypy/issues/6445
-# flags: --disallow-any-generics --no-new-semantic-analyzer
+# flags: --disallow-any-generics --new-semantic-analyzer
 from typing import NamedTuple
 
 N = NamedTuple('N', [('x', N)])  # type: ignore
 n: N
 [out]
 
 [case testCheckDisallowAnyGenericsTypedDict]
-# Crashes on new semantic analyzer: https://github.com/python/mypy/issues/6445
-# flags: --disallow-any-generics --no-new-semantic-analyzer
+# flags: --disallow-any-generics --new-semantic-analyzer
 from typing import Dict, Any, Optional
 from mypy_extensions import TypedDict
 
-VarsDict = Dict[str, Any]
-HostsDict = Dict[str, Optional[VarsDict]]
+VarsDict = Dict[str, Any]  # type: ignore
+HostsDict = Dict[str, Optional[VarsDict]]  # type: ignore
```

Comment:
Why these two ignores are needed? Both these aliases are not recursive and don't even have forward refs.

Reply:
Good catch! Yes, there is something suspicious going on.